util/TimeSeries.js

```diff
@@ -33,7 +33,33 @@ boneAPITimeSeries.granularities = {
   '1day'     : { ttl: boneAPITimeSeries.days(30) , duration: boneAPITimeSeries.days(1) },
 }
 
+const timeSeriesWithTz = new TimeSeries(rclient, "timedTrafficWithTz")
+timeSeriesWithTz.granularities = {
+  '1day': { ttl: timeSeries.weeks(52), duration: timeSeries.days(1) },
+  '1month': { ttl: timeSeries.months(24), duration: timeSeries.months(1) }
+}
+
+// set flag
+const timeSeriesWithTzBeginingTs = "time_series_with_tz_ts";
+(async () => {
+  const ts = await rclient.hgetAsync("sys:config", timeSeriesWithTzBeginingTs)
+  if (!ts) {
+    await rclient.hsetAsync("sys:config", timeSeriesWithTzBeginingTs, new Date() / 1000);
+  }
+})()
+
 module.exports = {
-  getTimeSeries: function() {return timeSeries},
-  getBoneAPITimeSeries: function() {return boneAPITimeSeries}
+  getTimeSeries: function () { return timeSeries },
+  getBoneAPITimeSeries: function () { return boneAPITimeSeries },
+  getTimeSeriesWithTz: function () { return timeSeriesWithTz },
+  supportTimeSeriesWithTz: async function () {
```

Comment:
keys is very costly, this check if one time only for each fireapi process, should not check for every api call. check once and use sys:config flag to store the check result.